packages/frontend/src/components/bridge/BridgeConfigEditor.tsx

```diff
@@ -29,28 +33,72 @@ ignoredProperties.forEach((prop) => {
 
 export interface BridgeConfigEditorProps {
   config: EditableBridgeConfig;
+  isValid: boolean;
   onChange: (
     config: EditableBridgeConfig | undefined,
     isValid: boolean,
   ) => void;
 }
 
+enum EditionModeType {
+  JSON_EDITOR = "JSON_EDITOR",
+  FIELDS_EDITOR = "FIELDS_EDITOR",
+}
+
 export const BridgeConfigEditor = ({
   config,
   onChange,
+  isValid,
 }: BridgeConfigEditorProps) => {
+  const [editionMode, setEditionMode] = useState(EditionModeType.FIELDS_EDITOR);
   const valueChanged = (event: {
     isValid: boolean;
     value?: EditableBridgeConfig;
   }) => {
     onChange(event.value, event.isValid);
   };
 
+  const handdleChangeEditionType = () => {
+    setEditionMode(
+      editionMode === EditionModeType.FIELDS_EDITOR
+        ? EditionModeType.JSON_EDITOR
+        : EditionModeType.FIELDS_EDITOR,
+    );
+  };
+
   return (
-    <JsonEditor
-      value={config}
-      onChange={valueChanged}
-      schema={editableConfigSchema}
-    />
+    <Box position="relative">
```

Comment:
instead of absolute positioning, you can make the box `display="flex"` and `justifyContent="space-between"`